src/components/serverNotifications/serverNotifications.js

```diff
@@ -197,30 +197,30 @@ define(['connectionManager', 'playbackManager', 'events', 'inputManager', 'focus
         events.on(apiClient, "message", onMessageReceived);
     }
 
-    function enableNativeGamepadKeyMapping() {
-        if (window.navigator && "string" == typeof window.navigator.gamepadInputEmulation) {
-            window.navigator.gamepadInputEmulation = "keyboard";
-            return true;
+    function attachgamepad(e) {
+        if (navigator.getGamepads.length > 0) {
+            require(["components/serverNotifications/gamepadtokey"]);
+            console.log("Gamepad connected! Attaching gamepadtokey.js script");
         }
-
-        return false;
     }
 
-    function isGamepadSupported() {
-        return "ongamepadconnected" in window || navigator.getGamepads || navigator.webkitGetGamepads;
+    function dettachgamepad(e) {
+        delete require.cache[require(["components/serverNotifications/gamepadtokey"])];
```

Comment:
I didn't tested this PR - just with my eyes. Should remained gamepads be checked before module unloading?

Another option: instead of loading/unloading the module, you can start/stop `runInputLoop`. Then all gamepad-related logic can be moved to `gamepad` module.